can/interfaces/usbtin.py

```diff
@@ -0,0 +1,77 @@
+"""
+Interface for the USBtin module
+
+Implementation references:
+* Uses the pyUSBTin API (https://github.com/fishpepper/pyUSBtin)
+* Interfaces with the USBtin module (https://www.fischl.de/usbtin/)
+"""
+
+from typing import List, Optional, Tuple, Any
+
+import logging
+import pyusbtin  # type: ignore
+import can
+
+logger = logging.getLogger(__name__)
+
+
+class USBtinBus(can.BusABC):
+    """
+    usbtin interface
+    """
+
+    def __init__(self, channel: str, baudrate: int):
```

Comment:
Shouldn't you capture all other arguments (`*args, **kwargs`) and forward them to `super`?